docs/guides/creatingasplitter.md

```diff
@@ -24,71 +24,105 @@ An event `Splitter` is part of the TriggerMesh routing solution. It has the simp
 
 Let's create the required objects:
 
-- [x] The `sockeye` target which serves as an event display.
-- [x] The `PingSource` which produces a JSON array in its payload.
-- [x] The `Splitter` to generate the multiple events.
+- [x] A `Broker` that will decouple the event producer and consumer and provide delivery guarantees, and its `Trigger` that will send events to the Splitter
+- [x] A `WebhookSource` which we'll use to produce an event with a JSON array in its payload.
+- [x] A `Splitter` to generate multiple events from the JSON payload.
```

Comment:
not a blocker. But i think links to the respective documentation for each component would be really nice in these sections.